gradle/patchVerifer.gradle

```diff
@@ -0,0 +1,194 @@
+import java.util.regex.Pattern
+
+def ignore = [
+        "patches/minecraft/net/minecraft/client/renderer/ViewFrustum.java.patch",
+        "patches/minecraft/net/minecraft/data/BlockModelDefinition.java.patch"
```

Comment:
These should verify, but also just delete the patch silently if the patches are ONLY the S2S artifacts.